csv_to_json.go

```diff
@@ -52,16 +52,16 @@ func csvToJSON(context *cli.Context) error {
 
 	var columns []string
 	for tick := 0; ; tick++ {
-		r, err := reader.Read()
```

Comment:
`err` is not shadowed here due to `if err == io.EOF` resulting to **true** will return early based on the following nested `if` condition.

Reply:
fair enough, I guess in that case, it might not have any side effect.
That said, if you refactor your code, you might not realized the error was indeed shadowed, whereas if you just create a diff err var, this will never happen, for sure.